spec/stylers/ui_label_styler.rb

```diff
@@ -21,6 +21,11 @@ def ui_label_centered(st)
     st.text_alignment = :centered
   end
 
+  def ui_label_text_align(st)
+    ui_label_kitchen_sink(st)
+    st.text_align = :left
```

Comment:
Should we check for `:center` or `:right` since the default is `:left`? I'm not sure this test is actually checking anything has changed when you specify `:left`.